.github/scripts/windows/test_task.bat

```diff
@@ -9,7 +7,7 @@ set NO_INTERACTION=1
 set REPORT_EXIT_STATUS=1
 set SKIP_IO_CAPTURE_TESTS=1
 
-call %~dp0find-target-branch.bat
+call %PHP_SRC_DIR%/find-target-branch.bat
```

Comment:
Not sure how this ever worked. `%~dp0` should be current directory and path. But on my local it was the path to this script. So I fixed it.